### ISSUE TITLE
speach hub should be aded to nvda sinth list

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3267]
**Reported by Animesh.mr on 2013-06-07 12:02**
speach hub is a tts server it may be used with nvda screenreader for more info go to  
http://www.speechhub.org/index.html

**Comment by leonardder** [comment]
[This](http://www.speakon.org.uk/SpeechHub.html) is not yet added on https://github.com/nvaccess/nvda/wiki/ExtraVoices

**Comment by Brian1Gaff** [comment]
Is this project still alive. I know the original still runs, but not seen 
anything from then in over a year.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by bhavyashah** [comment]
Regardless of whether or not this project is still alive, the original version of Speechhub for NVDA was quite popular and well-received. Could someone (CC at josephsl) please add a note about this on the Get Extra Voices page of the Github NVDA Wiki? This should be trivial to resolve.

**Comment by josephsl** [comment]
Hi, if the add-on is still being developed and if contacting the developer is still possible, then yes, it should be featured on our wiki. Thanks.

**Comment by Brian1Gaff** [comment]
I think development is over but the synth still works as indeed does Pico as 
well
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by ehollig** [comment]
It appears that SpeechHub can be downloaded from http://www.speakon.org.uk/SpeechHub.html. I am not sure if it is still being developed or not. I assume not. @Brian1Gaff do you know if this is still being developed? @josephsl do you think it is worth presenting this on the synth list on our community page if this addon is not being further developed?

**Comment by josephsl** [comment]
Hi, ultimately, it is up to developers to say if it is being developed, but given no updates (last time I checked), I think it is better to leave this out of synth page, especially once we start official Python 3 transition workflow. Thanks.

**Comment by ehollig** [comment]
Given this synth has not had any updates lately, I am going to close this issue.
